_includes/intro.html

```diff
@@ -16,9 +16,10 @@ <h2 class="section-headline">TC39</h2>
     <div class="g-col-4">
       <h2 class="section-headline">Contribute</h2>
       <p>
-        TC39 welcomes contributions from the JavaScript community, whether it is feedback on existing proposals, improved documentation,
-        testing, implementations, or even language feature ideas.
-        <!-- TODO: Make each of the things in the preceding sentence links to the appropriate guides -->
+       TC39 welcomes contributions. This includes proposal feedback, documentation, writing tests or
+       implementations, and language feature ideas.
```

Comment:
`This includes` sounds kind of like a complete list to me. If we're dropping `the JavaScript community` (_which is probably good in the interest of avoiding "us vs. them"_), maybe we could direct it right at the reader, like `You can help by...`?